sympy/functions/combinatorial/factorials.py

```diff
@@ -1057,3 +1058,86 @@ def _eval_is_nonnegative(self):
                 return True
             elif k.is_even is False:
                 return  False
+
+class multinomial(CombinatorialFunction):
+    r"""Implementation of the multinomial coefficient.
+        They can be expressed in numerous ways, including as a product of binomial coefficients or of factorials:
```

Comment:
This needs to be shortened with a reference given to wikipedia or some other reference for details.

Reply:
I've given the references towards the end of the docstring ... I'll shorten this part tho